.ci/heartbeat-synthetics.groovy

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env groovy
```

Comment:
This particular pipeline is similar to the .ci/packaging.groovy one, so I'd say to update .ci/packaging.groovy.

Reply:
I'm now running a packaging with the comment -> https://github.com/elastic/beats/pull/23767#issuecomment-770777607

In a quick view, I think it might be enough with the current packaging pipeline

Reply:
this is temporal, only until GA, it publishes on a separate namespace, when we'll be in GA the regular packaging will make the publishing of the Docker image without any modification

Reply:
in any case, I've asked because we will publish the same in two namespaces. The regular Heartbeat Docker image will have synthetics.

Reply:
Sooner than that! We can switch to the regular packaging in 7.12 (which is pre-GA). This is only for 7.11

Reply:
what I mean, is that we are publishing synthetics along with heartbeat `docker.elastic.co/beats/heartbeat:7.11.0` since the code is integrated on branch 7.11.

`.ci/heartbeat-synthetics.groovy` will public the same `docker.elastic.co/beats/heartbeat:7.11.0` but in other place `docker.elastic.co/experimental/heartbeat:7.11.0-synthetics`, it is exactly the same Docker image.